.github/workflows/release-new-version.yml

```diff
@@ -0,0 +1,122 @@
+name: Release new version
+run-name: New release for "${{ github.ref_name }}" triggered by ${{ github.actor }}
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: false
+        type: string
+      tag_suffix:
+        required: true
+        type: string
+      environment:
+        required: true
+        type: string
+
+env:
+  ECR_REPOSITORY: 'filplus-registry'
+
+jobs:
+  format_and_lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: '16.14.2'
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Run ESLint
+        run: npm run lint
+
+      - name: Run Prettier
+        run: npm run prettier
```

Comment:
This `format_and_lint` will now be executed twice - once for prod and once for staging. Maybe it should be moved to `deploy-new-version.yml` instead? Only build & push needs to happen twice